ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.9-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
```

Comment:
Same comments, edit commit message and squash into 5da16afcabbc3525eba8294f664029dd735990d9